### ISSUE TITLE
Which version of keras are you using? I get an error in output = dense(vocab_size, activation = 'softmax')(decoder_lstm) line in chatbot_training.py.

### ISSUE THREAD

**Issue by akhil2495** [sekharvth/chatbot-keras#1]

**Comment by sekharvth** [comment]
What is the exact error message you're getting?

**Comment by sekharvth** [comment]
Closing this issue for lack of activity. If the error persists, feel free to reopen.
